packages/flutter/lib/src/material/divider_theme.dart

```diff
@@ -0,0 +1,167 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:ui' show lerpDouble;
+
+import 'package:flutter/foundation.dart';
+import 'package:flutter/widgets.dart';
+
+import 'theme.dart';
+
+/// Defines the visual properties of [Divider], [VerticalDivider], and dividers
```

Comment:
Sadly, this expression comes up repeatedly, or probably needs to. Might be better to just use the same expression over and over rather than shortening it with etc or just leaving some of if out.

Reply:
done